app/src/main/java/com/android/sample/ui/composables/DateTimePicker.kt

```diff
@@ -46,17 +72,40 @@ fun DateTimePicker(
     datePicker.show()
   }
 
-  OutlinedTextField(
-      value =
-          selectedDateTime?.let {
-            "${it.dayOfMonth} ${it.month.name.lowercase().capitalize()} ${it.year} at ${it.hour}:${it.minute.toString().padStart(2, '0')}"
-          } ?: "Select Date and Time",
-      onValueChange = { /* No-op since we control the value */},
-      readOnly = true,
-      label = { Text(title) },
-      trailingIcon = {
-        IconButton(onClick = { showDateTimePickers() }) {
-          Icon(imageVector = Icons.Default.Edit, contentDescription = "Edit Date")
+  Column(modifier = Modifier.padding(vertical = 8.dp)) {
+    Text(
+        text = title,
+        style =
+            MaterialTheme.typography.bodySmall.copy(
+                color = TitleBlue,
+                fontWeight = FontWeight.Bold,
+                fontSize = FontSizes.SubtitleFontSize),
+        modifier = Modifier.padding(bottom = 4.dp))
+
+    Card(
+        shape = LeafShape,
+        colors = CardDefaults.cardColors(containerColor = LightBackground),
+        modifier =
+            Modifier.fillMaxWidth().height(Dimensions.ButtonHeight).clickable {
+              showDateTimePickers()
+            },
+        elevation = CardDefaults.cardElevation(4.dp)) {
+          Row(
+              verticalAlignment = Alignment.CenterVertically,
+              modifier = Modifier.fillMaxSize().padding(horizontal = Dimensions.LargePadding)) {
+                Text(
+                    text = displayText,
+                    style =
+                        MaterialTheme.typography.bodyMedium.copy(
+                            color = if (selectedDateTime == null) Color.Gray else Color.Black),
```

Comment:
You can use our color in Color.kt !

(You can use NeutralGray and Black)

Reply:
Done